test/TestApps/PushNotificationsDemoApp/main.cpp

```diff
@@ -92,21 +92,12 @@ winrt::Microsoft::Windows::PushNotifications::PushNotificationChannel RequestCha
 
 int main()
 {
-    if (Test::AppModel::IsPackagedProcess())
+    if (!Test::AppModel::IsPackagedProcess())
```

Comment:
why !Test::AppModel::IsPackagedProcess()?  code under if looks like you are creating activationinfo from guid. Please check?

Reply:
This is the bootstrapper check which only applies to unpackaged.